DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionSwizzler.swift

```diff
@@ -0,0 +1,273 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+internal class URLSessionSwizzler {
```

Comment:
This swizzler differs from the one currently implemented in v1, it only expose static accessors.